3542/1/lib/repo.rb

```diff
@@ -0,0 +1,63 @@
+require 'httparty'
+require 'nokogiri'
+require 'open-uri'
+
+#:reek:TooManyInstanceVariables
+class Repo
+  URI = 'https://api.github.com/search/repositories'.freeze
+
+  attr_reader :name, :used_by, :watched_by, :stars, :forks, :contributors, :issues
+
+  # rubocop:disable Metrics/AbcSize
+  # rubocop:disable Security/Open
+  def initialize(gem_name)
+    @api_response = HTTParty.get(URI, query: { q: gem_name })
+    @repo = repository
```

Comment:
This instance variable is overhead.
You can define the rest of necessary instance variables directly in the method `repository`